catalyst-gateway/tests/Earthfile

```diff
@@ -1,26 +1,89 @@
 VERSION 0.7
 
+build:
```

Comment:
I thought you mentioned not to use the artifact from the main earthfile, so that all the build checks and tests would not be run again, but I probably misunderstood you